package-lock.json

```diff
@@ -19,9 +19,9 @@
       "integrity": "sha512-UFMC4ZeFC48Tpvj7C8UgLvtkaUuovQX+5xNWrsIoMG8o2z+XFKjKaN9iVmS84dPwVN00W4wPmqvYoZF3EGAsfw==",
       "dev": true,
       "requires": {
-        "chalk": "2.4.1",
-        "esutils": "2.0.2",
-        "js-tokens": "4.0.0"
+        "chalk": "^2.0.0",
```

Comment:
I find odd that we are getting older versions instead of recent ones. Is this on purpose?

Reply:
It's not an older version, the lock file format in newer versions of npm just now changed from showing exact versions in this section to showing the version range.

2.4.1 is compatible with ^2.0.0

Reply:
After merged though we probably should have everyone upgrade the npm version

Reply:
All right! Thanks a lot for the clarification.